systems/plants/RigidBodyManipulator.cpp

```diff
@@ -535,9 +535,32 @@ void RigidBodyManipulator::updateCollisionElements(const shared_ptr<RigidBody>&
     } else {
       collision_model->updateElementWorldTransform(*id_iter, body->T);
     }
+    
+    // update the body's contact points    
+    Matrix3Xd contact_pts;
+    getTerrainPoints(*body, contact_pts);
+    body->contact_pts = contact_pts.colwise().homogeneous();
```

Comment:
This is somewhat hacky. `RigidBody::contact_pts` is a `MatrixXd`, but resized to 4 rows in `constructModelmex.cpp`. Any advice?